packages/dids/tests/methods/did-dht.spec.ts

```diff
@@ -1153,10 +1160,35 @@ describe('DidDhtDocument', () => {
 
   describe('Web5TestVectorsDidDht', () => {
     it('resolve', async () => {
-      for (const vector of DidDhtResolveTestVector.vectors as any[]) {
+      for (const vector of resolveTestVectors.vectors as any[]) {
         const didResolutionResult = await DidDht.resolve(vector.input.didUri);
         expect(didResolutionResult.didResolutionMetadata.error).to.equal(vector.output.didResolutionMetadata.error);
       }
     }).timeout(30000); // Set timeout to 30 seconds for this test for did:dht resolution timeout test
   });
 });
+
+// vectors come from https://did-dht.com/#test-vectors
+describe('Official DID:DHT Vector tests', () => {
```

Comment:
ok this needs to be the other way,
Our test vectors look in a hardcoded spot 

Web5TestVectorsDidDht.resolve()
<img width="394" alt="image" src="https://github.com/TBD54566975/web5-js/assets/5314059/21a7f054-dbf6-4ed6-9127-77569216cda5">

so keep the resolve() one the same with the web5-spec test vectors, and then make a new one called something like:

describe('Local DID:DHT resolve vectors, () => {

Reply:
Not sure I get what you mean, let's chat!

Reply:
Oh just a rename, good to go!